### PR TITLE
Image processing with photo upload

### PR DESCRIPTION
**Pull request by animeshgarai09**
Created flow for posting images for my social media
User can drag and drop multiple images ---> User can crop and reposition images ---> Images get cropped and resized for reduced file size ---> user can write a caption for the post.
https://felix-social-react-git-image-processing-animeshgarai09.vercel.app/home

### CODE REVIEW

File: src/components/image-crop/image-crop.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
+import styles from "./image-crop.module.scss"
+import Cropper from 'react-easy-crop'
+import { useState, useCallback, useEffect } from "react"
+import useImageDimension from "../../hooks/useImageDimension"
+import useImageResize from "../../hooks/useImageResize"
```
Comment (VLeads) [inline src/components/image-crop/image-crop.js:5]: Found this file code new to me. Will look deeper into it. Doing great work.

File: src/hooks/useImageResize.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    quality = 80, // Is the quality of the resized new image.
+    rotation = 0, // Is the degree of clockwise rotation to apply to uploaded image.
+    outputType = "base64", // Is the output type of the resized new image.
+}) => {
+    return (file) =>
+        new Promise((resolve) => {
+            Resizer.imageFileResizer(
+                file,
+                maxWidth,
+                maxHeight,
```
Comment (VLeads) [inline src/hooks/useImageResize.js:16]: It's great that you are compressing image.

**Review by VLeads — APPROVED** [review]
Overall code looks good. There were some things that were new to me. You are good to go !!

**Review by b0llu — APPROVED** [review]
Really well written code was not able to find anything to suggest! Great work you can merge this PR

**Pull request merged**